targets/TARGET_Ambiq_Micro/TARGET_Apollo3/sdk/mcu/apollo3/hal/am_hal_uart.c

```diff
@@ -436,11 +436,11 @@ am_hal_uart_configure(void *pHandle, const am_hal_uart_config_t *psConfig)
     //
     // Set up any buffers that might exist.
     //
-    buffer_configure(pHandle,
-                     psConfig->pui8TxBuffer,
-                     psConfig->ui32TxBufferSize,
-                     psConfig->pui8RxBuffer,
-                     psConfig->ui32RxBufferSize);
+    // buffer_configure(pHandle,
+    //                  psConfig->pui8TxBuffer,
+    //                  psConfig->ui32TxBufferSize,
+    //                  psConfig->pui8RxBuffer,
+    //                  psConfig->ui32RxBufferSize);
```

Comment:
was it necessary to change the HAL here? is it not enough to simply set these 4 buffers as NULL? (i figure the HAL would not do anything with them if they were not provided)